### ISSUE TITLE
Add a newline after custom summary

### ISSUE THREAD

**Issue by mikeerickson** [clessg/progress-bar-webpack-plugin#8]
Was playing with the customSummary option and ran into this

```
[14:03:32] merickson:angular-sandbox $ webpack
  build [====================] 100% (21.2 seconds)

This goes here!(21.186s)Hash: 6087bed5bc21e5cf657e
Version: webpack 1.13.0
```

Seems a new line should be between the customSummary message and time

I can PR this later if you wish.  Surely nothing to stop progress, but you know, appearance is important :-)

**Comment by clessg** [comment]
Oh man, this is a disastrous bug.

Thanks for reporting this. If you could PR it, I would be grateful!

**Comment by mikeerickson** [comment]
@clessg will take care of it tomorrow :-)

**Comment by e-cloud** [comment]
@mikeerickson what's the progress of this issue

**Comment by mikeerickson** [comment]
Whoops, let that slip through the cracks.  I will take care of this later today 😜

**Comment by clessg** [comment]
No worries, I understand life can be busy. I went ahead and fixed it. Published as 1.9.0. Thanks for informing us of the issue!

**Comment by mikeerickson** [comment]
@clessg Sorry about delay and failing this mission.  But thx for addressing nonetheless :-)
